xml/cioos_template.xml

```diff
@@ -273,56 +495,61 @@
       <!-- platform: mandatory -->
       <mac:platform>
         <mac:MI_Platform>
-          <!-- description: mandatory -->
-          <mac:description>
-            <gco:CharacterString>
-              <!-- CIOOS core mandatory element -->
-              <!-- MI_Metadata/acquisitionInformation/MI_AcquisitionInformation/platform/MI_Platform/description/CharacterString -->
-            </gco:CharacterString>
-          </mac:description>
+
           <!-- identifier: mandatory -->
           <mac:identifier>
             <mcc:MD_Identifier>
               <!-- authority: mandatory -->
               <mcc:authority>
                 <cit:CI_Citation>
+                  <!-- validation says cit:CI_Citation requires title -->
+                  <cit:title></cit:title>
+
                   <!-- citedResponsibleParty: mandatory -->
                   <cit:citedResponsibleParty>
                     <cit:CI_Responsibility>
                       <!-- role: mandatory -->
                       <cit:role>
-                        <cit:CI_RoleCode>
+                        <cit:CI_RoleCode codeList="http://standards.iso.org/iso/19115/resources/Codelists/cat/codelists.xml#CI_RoleCode" codeListValue="{{ record['platform_role'] }}">
                           <!-- CIOOS core mandatory element -->
                           <!-- MI_Metadata/acquisitionInformation/MI_AcquisitionInformation/platform/MI_Platform/identifier/MD_Identifier/authority/CI_Citation/citedResponsibleParty/CI_Responsibility/role/CI_RoleCode -->
+
                         </cit:CI_RoleCode>
                       </cit:role>
                       <!-- party: mandatory -->
                       <cit:party>
-                        <cit:CI_Party>
+                        <cit:CI_Organisation>
```

Comment:
Eventually we'll need more Jinja logic here to check whether a CI_Organisation or a CI_Individual is needed here.